tests/services/fichiers.service.test.ts

```diff
@@ -0,0 +1,34 @@
+import { ApiClient } from '../../clients/api.client'
```

Comment:
```suggestion
import { ApiClient } from 'clients/api.client'
```
pour tous les imports